src/lib/ng-package/entry-point/compile-ngc.transform.ts

```diff
@@ -24,31 +26,40 @@ export const compileNgcTransformFactory = (StylesheetProcessor: typeof Styleshee
       // Compile TypeScript sources
       const { esm2015, declarations } = entryPoint.data.destinationFiles;
       const { basePath, cssUrl, styleIncludePaths } = entryPoint.data.entryPoint;
-      const { moduleResolutionCache, ngccProcessingCache, stylesheetProcessor = new StylesheetProcessor(basePath, cssUrl, styleIncludePaths) } = entryPoint.cache;
-      entryPoint.cache.stylesheetProcessor = stylesheetProcessor;
+      const { moduleResolutionCache, ngccProcessingCache } = entryPoint.cache;
 
-      const ngccProcessor = tsConfig.options.enableIvy
-        ? new NgccProcessor(ngccProcessingCache, tsConfig.project, tsConfig.options, entryPoints)
-        : undefined;
+      if (tsConfig.options.enableIvy !== false) {
```

Comment:
Maybe it would be wise to have an environment variable to disable the new pipeline; to allow people to fallback on the legacy pipeline if any issues pop up.

Reply:
Yes agreeded.